tests/GridFS/FunctionalTestCase.php

```diff
@@ -53,7 +53,7 @@ protected function assertStreamContents(string $expectedContents, $stream): void
      *
      * @return resource
      */
-    protected function createStream(string $data = '')
+    protected static function createStream(string $data = '')
```

Comment:
Technically not a data provider but there's also no reason this method can't be static :+1:

Reply:
I made it static because It's used in a data provider.